src/platforms/common/enriching-events/breadcrumbs.mdx

```diff
@@ -25,14 +25,16 @@ You can manually add breadcrumbs whenever something interesting happens. For exa
 
 <PlatformContent includePath="breadcrumbs-example" />
 
-## Automatic Breadcrumbs
+The available breadcrumb keys are "type", "category", "message", "level", "timestamp" (which many SDKs will set automatically for you), and "data", which is the place to put any information you'd like the breadcrumb to include. Using keys other than these six won't cause an error, but will result in the data being dropped when the event is processed by Sentry.
```

Comment:
We may want to call out that this is relevant mostly for untyped languages like JavaScript. In all typed languages, Breadcrumbs and their fields are typed out, too. Maybe, this could be shown conditionally based on the platform? 

Should we also wrap the field names in code?

Reply:
The data inside breadcrumbs can also be partially typed. The canonical documentation for this lives at:
https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/

Reply:
> We may want to call out that this is relevant mostly for untyped languages like JavaScript. In all typed languages, Breadcrumbs and their fields are typed out, too. Maybe, this could be shown conditionally based on the platform?

Oh, right, that makes sense. So it would be shown for JS/Node/RN, Python, PHP, Ruby, Elixir, and Perl, but not for Java, Native, C#, Dart, Go, or Rust? What about "Apple," which is both Obj-C (dynamically typed) and Swift (statically typed)?

Reply:
If something like this is possible, yes. 

Only exception: For native, we can show it since we haven't typed out the protocol.

Reply:
Okay, should be good now.